tasks/ui_framework_test.js

```diff
@@ -0,0 +1,36 @@
+const jest = require(`jest`);
```

Comment:
What's with the mix of backticks and single quotes in this file?  We should stick to single quotes when not interpolating per the styleguide.